db/migrate/data/20160127210624_convert_configurations_to_settings_changes/vmdb.tmpl.yml

```diff
@@ -332,6 +332,8 @@ workers:
         :amqp_heartbeat: 30
       :event_catcher_amazon:
         :poll: 15.seconds
+      :event_catcher_hawkular:
+        :poll: 15.seconds
```

Comment:
This is the tail of $mw_log, not a grep.  But you are correct.  Looking at the core code, it seems monitoring is supposed to invoked and handed off to the provider.  Because the thread is exiting and restarting (I confirmed in evm.log) it looked very much to me like monitor_events was just being invoked on a schedule.  I will move to the approach we discussed. Thanks for digging into this confusion.